docker/Dockerfile.linux.ppc64le

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3 as alpine
+RUN apk add -U --no-cache ca-certificates
+
+RUN wget https://github.com/isuruf/tmate/releases/download/2.4.0/tmate-2.4.0-static-linux-ppc64le.tar.xz
```

Comment:
@bradrydzewski, FYI, this is from my github repo because there was no official release.

Reply:
>     * Ubuntu: https://packages.ubuntu.com/hirsute/gcc => ppc64el
> 
>     * Fedora's build system. : https://koji.fedoraproject.org/koji/buildinfo?buildID=1708867 => ppc64le
> 
> 
> The `uname -m` might show the good candidate name for PPC little endian.

ubuntu@travis-bastion:~$ uname -a
Linux travis-bastion 4.15.0-137-generic #141-Ubuntu SMP Fri Feb 19 13:44:14 UTC 2021 ppc64le ppc64le ppc64le GNU/Linux